composer.json

```diff
@@ -27,7 +27,8 @@
         "symfony/options-resolver": "^2.3 || ^3.0",
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
         "sonata-project/admin-bundle": "^3.0",
-        "sonata-project/datagrid-bundle": "^2.2"
+        "sonata-project/datagrid-bundle": "^2.2",
+        "cocur/slugify": "^1.4 || ^2.0"
```

Comment:
Thanks! Sadly, my build is stuck, I emailed Travis about that, but no answer so far.